docs/src/pages/versions/StableVersions.js

```diff
@@ -43,42 +45,51 @@ async function getBranches() {
 
 class StableVersions extends React.Component {
   state = {
-    versions: [],
+    docs: [],
   };
 
   componentDidMount = async () => {
     const branches = await getBranches();
-    const versions = branches.map(n => n.name);
-    versions.reverse().pop(); // most recent first & remove 'latest'
-    versions.push('v0.20.1');
-
-    this.setState({ versions });
+    let docs = branches.map(n => n.name);
+    docs.sort().reverse();
```

Comment:
The GitHub API returns the branches already sorted (regardless of the order they were added), so while harmless, `sort()` won't actually do anything here as I understand it.

Reply:
> The GitHub API returns the branches already sorted

Is this by design or just by luck?

Reply:
It appears to be by design - I tested it every which way, but keeping `sort()` can't do any harm.